index.html

```diff
@@ -110,8 +111,430 @@ <h3 class="title title_level-3">An Amazing Feature</h3>
+                                        The gentlemen who rented the room would sometimes take their evening meal at home in
+                                        the living.
+                                    </p>
+                                    <div>
+                                        <span class="articles__person-card__img"></span>
+                                        <div class="articles__person-info">
+                                            <p class="articles__person-name">ANNIE MALKOVICH</p>
+                                            <p class="articles__person-date">2017/10/16</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </section>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            <!--CLIENTS-->
+            <section class="substrate">
+                <div class="container">
+                    <div class="clients">
+                        <div class="row justify-content-center">
+                            <div class="header clients__header">
+                                <h2 class="title title_level-2">Clients</h2>
+                            </div>
+                        </div>
+                        <div>
+                            <div class="row justify-content-center">
+                                <span class="img clients__img"></span>
+                                <span class="img clients__img"></span>
+                                <span class="img clients__img"></span>
+                                <span class="img clients__img"></span>
+                                <span class="img clients__img"></span>
+                                <span class="img clients__img"></span>
+                                <div class="fa fa-angle-right fa-4x arrow clients__arrow"></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            <!--PORTFOLIO-->
+            <section class="container">
+                <div class="portfolio">
+                    <div class="row justify-content-center">
+                        <div class="col-8">
+                            <div class="header portfolio__header">
+                                <h2 class="title title_level-2">Portfolio & Screenshot</h2>
+                                <p class="text text_level-2">
+                                    The gentlemen who rented the room would sometimes take their evening meal at home in the
+                                    living room that was used by everyone...
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <hr class="underline portfolio__underline">
```

Comment:
Почему? `<hr>` семантически же это линия/черта.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

> The **HTML `<hr>` element** represents a thematic break between paragraph-level elements (for example, a change of scene in a story, or a shift of topic with a section); historically, this has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS.

Reply:
О как! Спасибо)